src/main/deploy/parameters.toml

```diff
@@ -46,4 +46,24 @@ kD = 0.0
 left_front_offset = 210.0
 left_back_offset = 79.0
 right_front_offset = 9.2
-right_back_offset = 330.0
\ No newline at end of file
+right_back_offset = 330.0
+
+[arm.lift_pid]
+kP = 0.1
+kI = 0
+kD = 0
+
+[arm.extend_pid]
+kP = 0.1
+kI = 0
+kD = 0 
```

Comment:
I'm pretty surprised that only P control is being used here - does that work well?

Reply:
after testing it tonight, yeah actually. It's very slow to catch up to the setpoint right now--I think I have an idea on what to do about that that I'll write up in mattermost or something